src/Uno.UWP/Helpers/AppDataUriEvaluator.cs

```diff
@@ -9,7 +9,7 @@ namespace Uno.Helpers
 	/// <summary>
 	/// Logic is based on https://docs.microsoft.com/en-us/windows/uwp/app-resources/uri-schemes#ms-appdata.
 	/// </summary>
-	internal static class AppDataUriEvaluator
```

Comment:
Let's not make this public. Let's use InternalsVisibleTo if possible, otherwise, we can include that file again in the projects that need it.

Reply:
I agree and was not sure if there was another way to avoid that.
Thank you for sharing InternalsVisibleTo :) Was not aware of it